src/testshade/cuda/optix_grid_renderer.cu

```diff
@@ -115,6 +115,20 @@ extern "C" __global__ void __anyhit__()
     // do nothing
 }
 
+extern "C" __global__ void __raygen__setglobals()
+{
+
+    // Set global variables
+    OSL::pvt::osl_printf_buffer_start    = render_params.osl_printf_buffer_start;
+    OSL::pvt::osl_printf_buffer_end      = render_params.osl_printf_buffer_end;
+    OSL::pvt::s_color_system             = render_params.color_system;
+    OSL::pvt::test_str_1                 = render_params.test_str_1;
+    OSL::pvt::test_str_2                 = render_params.test_str_2;
+}
+extern "C" __global__ void __miss__setglobals() { }
+extern "C" __global__ void __closesthit__setglobals() { }
+extern "C" __global__ void __anyhit__setglobals() { }
```

Comment:
Do you anticipate different sets of global various for the different program types?

Reply:
Nope -- I just filled them in in case OptiX expects those to exist (even if they don't do anything).  I'll look into removing them if it shouldn't cause any issues.